packages/legend-query-builder/src/components/explorer/QueryBuilderPropertySearchPanel.tsx

```diff
@@ -56,57 +60,160 @@ import {
 } from '../../stores/explorer/QueryBuilderExplorerState.js';
 import type { QueryBuilderState } from '../../stores/QueryBuilderState.js';
 import {
+  QUERY_BUILDER_EXPLORER_SEARCH_INPUT_NAME,
   QueryBuilderSubclassInfoTooltip,
   renderPropertyTypeIcon,
 } from './QueryBuilderExplorerPanel.js';
 import { QueryBuilderPropertyInfoTooltip } from '../shared/QueryBuilderPropertyInfoTooltip.js';
 import { QUERY_BUILDER_TEST_ID } from '../../__lib__/QueryBuilderTesting.js';
-import { FuzzySearchAdvancedConfigMenu } from '@finos/legend-lego/application';
+import { DocumentationLink } from '@finos/legend-lego/application';
+import { LEGEND_APPLICATION_DOCUMENTATION_KEY } from '@finos/legend-application';
 
-const prettyPropertyNameFromNodeId = (name: string): string => {
-  let propNameArray = name.split('.');
-  propNameArray = propNameArray.map(prettyCONSTName);
-  let propName = '';
-  propNameArray.forEach((p) => {
-    propName = `${propName + p}/`;
-  });
-  propName = propName.slice(0, -1);
-  return propName;
-};
+export const prettyPropertyNameFromNodeId = (
+  name: string,
+  spaceBetweenSlash?: boolean,
+): string =>
+  name
+    .split('@')
+    .map((p) => p.replace(/.*::/, ''))
```

Comment:
for `::` maybe we could use ELEMENT_PATH_DELIMITER

Reply:
Done!